### ISSUE TITLE
Failed to start process when deploying repo with nested meteor-app directory

### ISSUE THREAD

**Issue by angelcabo** [AdmitHub/meteor-buildpack-horse#34]
Trying to deploy a project with a meteor app nested in a directory called app/. It seems like the compile script knows to look for this type of setup (line 33):
```bash
[ ! -d "$APP_CHECKOUT_DIR/.meteor" ] && APP_CHECKOUT_DIR="$APP_CHECKOUT_DIR/app"
```
But then the app crashes with the following message in the log:

> 2015-07-03T20:39:40.677216+00:00 heroku[web.1]: Starting process with command '.meteor/heroku_build/bin/node .meteor/heroku_build/app/main.js'
> 2015-07-03T20:39:42.577897+00:00 app[web.1]: bash: .meteor/heroku_build/bin/node: No such file or directory

Could the issue be that the release script needs to take into account that the meteor app might be nested in an 'app' directory?

```bash
web: .meteor/heroku_build/bin/node .meteor/heroku_build/app/app/main.js
```

**Comment by AngelCabo** [comment]
Sorry, just realized this issue should be fixed with pr #23

**Comment by yourcelf** [comment]
PR is now merged -- so this should work correctly with master now.
